frontend/src/pages/wifiPage/connectDialog/ConnectDialog.tsx

```diff
@@ -102,6 +104,12 @@ export default ({
           <span className={styles.error}>
             There was an error connecting to {ssid}... please check your
             password and try again
+            {disconnectedFromAp && (
+              <>
+              <br></br>
+              Your computer has disconnected from the wifi network 'pi-top'. Please reconnect to it.
+              </>
+            )}
```

Comment:
When disconnectedFromAp is true we don't actually know that connecting to the ssid was unsuccessful. Plus having both messages at once is a lot of text.

```suggestion
          <span className={styles.error}>
            {disconnectedFromAp ? (
              <>
                Your computer has disconnected from the wifi network
                'pi-top-XXX'. Please reconnect to it.
              </>
            ) : (
              <>
                There was an error connecting to {ssid}... please check your
                password and try again
              </>
            )}
```